src/test/java/com/artipie/YamlSettingsTest.java

```diff
@@ -56,14 +57,12 @@ public void shouldSetFlatAsDefaultLayout() throws Exception {
             String.join(
                 "",
                 "meta:\n",
-                "  storage:\n",
-                "    type: s3\n",
-                "    path: /var/artipie/configs\n"
+                "  storage:\n"
```

Comment:
@IlyaMoskva and is `storage:` part really needed? We do not access `storage` in this test and I believe the string could be simplified to just `"meta:\n"

Reply:
It breaks in constructor or in other place before. Just 'meta:' doesn't work for me.

Reply:
@IlyaMoskva that's weird, there is just assignment in constructor. Could you please elaborate the problem?

Reply:
@olegmoz Exactly storage is not necessary here, but empty meta throws Exception in .yamlMapping(YamlSettings.KEY_META) call. It must contain at least something inside. I believe, it's out of scope for this ticket.

Reply:
@IlyaMoskva got it, thanks